tsl/test/expected/compression_ddl.out

```diff
@@ -1855,3 +1855,280 @@ ORDER BY device_id;
+(1 row)
+
+insert into test_partials values
+-- chunk1
+('2020-01-01 00:00'::timestamptz, 1, 2),
+('2020-01-01 00:01'::timestamptz, 2, 2),
+('2020-01-01 00:04'::timestamptz, 1, 2),
+-- chunk2
+('2021-01-01 00:00'::timestamptz, 1, 2),
+('2021-01-01 00:04'::timestamptz, 1, 2),
+-- chunk3
+('2022-01-01 00:00'::timestamptz, 1, 2),
+('2022-01-01 00:04'::timestamptz, 1, 2);
+-- enable compression, compress all chunks
+alter table test_partials set (timescaledb.compress);
+select compress_chunk(show_chunks('test_partials'));
+              compress_chunk               
+-------------------------------------------
+ _timescaledb_internal._hyper_34_117_chunk
+ _timescaledb_internal._hyper_34_118_chunk
+ _timescaledb_internal._hyper_34_119_chunk
+(3 rows)
+
+-- fully compressed
+explain (costs off) select * from test_partials order by time;
+                                   QUERY PLAN                                   
+--------------------------------------------------------------------------------
+ Custom Scan (ChunkAppend) on test_partials
+   Order: test_partials."time"
+   ->  Custom Scan (DecompressChunk) on _hyper_34_117_chunk
+         ->  Sort
+               Sort Key: compress_hyper_35_120_chunk._ts_meta_sequence_num DESC
+               ->  Seq Scan on compress_hyper_35_120_chunk
+   ->  Custom Scan (DecompressChunk) on _hyper_34_118_chunk
+         ->  Sort
+               Sort Key: compress_hyper_35_121_chunk._ts_meta_sequence_num DESC
+               ->  Seq Scan on compress_hyper_35_121_chunk
+   ->  Custom Scan (DecompressChunk) on _hyper_34_119_chunk
+         ->  Sort
+               Sort Key: compress_hyper_35_122_chunk._ts_meta_sequence_num DESC
+               ->  Seq Scan on compress_hyper_35_122_chunk
+(14 rows)
+
+-- test P, F, F
+insert into test_partials values ('2020-01-01 00:03', 1, 2);
+explain (costs off) select * from test_partials order by time;
+                                   QUERY PLAN                                   
+--------------------------------------------------------------------------------
+ Custom Scan (ChunkAppend) on test_partials
+   Order: test_partials."time"
+   ->  Merge Append
+         Sort Key: _hyper_34_117_chunk."time"
+         ->  Sort
+               Sort Key: _hyper_34_117_chunk."time"
+               ->  Custom Scan (DecompressChunk) on _hyper_34_117_chunk
+                     ->  Seq Scan on compress_hyper_35_120_chunk
+         ->  Sort
+               Sort Key: _hyper_34_117_chunk."time"
+               ->  Seq Scan on _hyper_34_117_chunk
+   ->  Custom Scan (DecompressChunk) on _hyper_34_118_chunk
+         ->  Sort
+               Sort Key: compress_hyper_35_121_chunk._ts_meta_sequence_num DESC
+               ->  Seq Scan on compress_hyper_35_121_chunk
+   ->  Custom Scan (DecompressChunk) on _hyper_34_119_chunk
+         ->  Sort
+               Sort Key: compress_hyper_35_122_chunk._ts_meta_sequence_num DESC
+               ->  Seq Scan on compress_hyper_35_122_chunk
```

Comment:
I think the first Sort is not necessary. 
Shouldnt the plan look like this:
```
 Custom Scan (ChunkAppend) on test_partials
   Order: test_partials."time"
   ->  Merge Append
         Sort Key: _hyper_34_117_chunk."time"
         ->  Custom Scan (DecompressChunk) on _hyper_34_117_chunk
               ->  Sort
                     Sort Key: compress_hyper_35_120_chunk._ts_meta_sequence_num DESC
                     ->  Seq Scan on compress_hyper_35_120_chunk
         ->  Sort
               Sort Key: _hyper_34_117_chunk."time"
               ->  Seq Scan on _hyper_34_117_chunk
   ->  Custom Scan (DecompressChunk) on _hyper_34_118_chunk
         ->  Sort
               Sort Key: compress_hyper_35_121_chunk._ts_meta_sequence_num DESC
               ->  Seq Scan on compress_hyper_35_121_chunk
   ->  Custom Scan (DecompressChunk) on _hyper_34_119_chunk
         ->  Sort
               Sort Key: compress_hyper_35_122_chunk._ts_meta_sequence_num DESC
               ->  Seq Scan on compress_hyper_35_122_chunk
```

Reply:
fixed now